### PR TITLE
css and layout update (issue #318)

### PR DESCRIPTION
**Pull request by diegosanmartino**

### CODE REVIEW

File: frontend/assets/application/conoscenze_ambientali/css/skin-home.css [hunk 1]
```diff
@@ -1,10 +1,10 @@
-.home {
+body {
```
Comment (mbarto) [inline frontend/assets/application/conoscenze_ambientali/css/skin-home.css]: did you check that the map page is not affected by this change?

File: frontend/assets/application/conoscenze_ambientali/css/skin-home.css [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
     background:url("images/desk2.jpg") no-repeat scroll center top #f2f2f2;
 	/*background-size:100%;*/
 	}
 
 @media (max-width: 767px) {
-.home {
+body {
```
Comment (mbarto) [inline frontend/assets/application/conoscenze_ambientali/css/skin-home.css]: did you check that the map page is not affected by this change?

File: frontend/js/components/MosaicTile.jsx [hunk 3]
```diff
@@ -24,12 +24,7 @@ const MosaicTile = React.createClass({
     getDefaultProps() {
         return {
            useLink: true,
-           boxStyle: {
-                backgroundColor: '#000',
-                backgroundRepeat: 'no-repeat',
-                backgroundPosition: 'center',
-                paddingTop: '130px'
-           }
+           boxStyle: { }
```
Comment (mbarto) [inline frontend/js/components/MosaicTile.jsx]: Did you check that the catalog in map page is not broken by this?

File: frontend/assets/application/conoscenze_ambientali/css/skin-home.css (hunk 2 above)
Comment (diegosanmartino) [inline frontend/assets/application/conoscenze_ambientali/css/skin-home.css]: a quick check... Are there problems in the map page?

Comment (mbarto) [inline frontend/assets/application/conoscenze_ambientali/css/skin-home.css]: In general, if you change a rule on body you can affect many things, so I want to be sure you checked before merging

Comment (diegosanmartino) [inline frontend/assets/application/conoscenze_ambientali/css/skin-home.css]: can we make a home.hml similar to map.html?

**Pull request merged**